src/lib/Libecl/ecl_verify.c

```diff
@@ -630,14 +630,49 @@ static struct ecl_attribute_def * ecl_find_attr_in_def(
 
 	if (attr_def) {
 		for (index = 0; index < limit; index++) {
-			if (!strcasecmp(attr_def[index].at_name, name))
-				return (&attr_def[index]);
+			char *pc = NULL;
+
+			if (strncasecmp(name, attr_def[index].at_name,
+					strlen(attr_def[index].at_name)) == 0) {
+				pc = name + strlen(attr_def[index].at_name);
+				if ((*pc == '\0') || (*pc == '.') || (*pc == ','))
```

Comment:
I don't understand the comma.  The period and null make sense.  Are you planning on passing a comma separated list into this function?